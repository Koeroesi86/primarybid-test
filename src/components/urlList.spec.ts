import { render } from '@testing-library/vue';
import ShortUrlResponse from '../types/ShortUrlResponse';
import UrlList from './urlList';

describe('UrlList', () => {
  it('should render', () => {
    const result = render(UrlList);

    expect(result.container).toMatchSnapshot();
  });

  it('should render list', () => {
    const result = render(UrlList, {
      props: {
        listing: [
          {
            url: 'https://example.com/asdf',
            original: 'https://something.com/example/url',
          },
        ] as ShortUrlResponse[],
      },
    });

    expect(result.container).toMatchSnapshot();
  });
});
