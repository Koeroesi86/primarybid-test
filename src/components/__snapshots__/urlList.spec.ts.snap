// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UrlList should render 1`] = `
<div>
  <div
    class="urlList"
  >
     
    <span>
      
        No urls yet. Why not create one?
      
    </span>
  </div>
</div>
`;

exports[`UrlList should render list 1`] = `
<div>
  <div
    class="urlList"
  >
    <a
      href="https://example.com/asdf"
      target="_blank"
      title="https://something.com/example/url"
    >
      
        https://example.com/asdf
      
    </a>
     
    <!---->
  </div>
</div>
`;
